features/nanostack/sal-stack-nanostack/nanostack/ws_management_api.h

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2018, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * \file ws_management_if.h
+ * \brief Wi-SUN management interface.
+ *
+ * This interface is used for configuring Wi-SUN devices.
+ * After creating the Wi-SUN interface, you can use this interface to configure the Wi-SUN device
+ * behaviour. When you are done with the configurations, you need to call interface up to enable a Wi-SUN node.
+ *
+ */
+
+#ifndef WS_MANAGEMENT_API_H_
+#define WS_MANAGEMENT_API_H_
+
+#include "ns_types.h"
+#include "net_interface.h" /* Declaration for channel_list_s. */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Regulatory domain values*/
+#define REG_DOMAIN_WW   0x00 // World wide
```

Comment:
This is obviously for Nanostack, but really all these defines should have `WS_` prefixes.